lms/static/sass/views/_account-settings.scss

```diff
@@ -0,0 +1,107 @@
+.account-settings-container {
+    @extend .container;
+    padding-top: $baseline*2;
+
+    > div {
+      background-color: transparent;
+      border: none;
+      box-shadow: none;
+    }
+
+    .ui-loading-indicator {
+         @extend .ui-loading-base;
+
+         padding-bottom: $baseline;
+
+         // center horizontally
+         @include margin-left(auto);
+         @include margin-right(auto);
+         width: $baseline*5;
```

Comment:
anytime there's a math operation on a css variable, our convention is to wrap these in parentheses. ($baseline*5)